### PR TITLE
feat(Product Stocks): build admin #index endpoint

### LINKED ISSUE #47: Need a new `product_stocks#create` which receives the following body
```json
{
    "quantity": 5,
    "toppings": [
        {
            "key": "color",
            "value": "white",
            "price_change": "+100"
        }, {
            "key": "size",
            "value": "XL",
            "price_change": "+10"
        }
    ]
}
```

### LINKED ISSUE #48: Need a new `product_stocks#index` that returns all the product stocks with its corresponding toppings

### PR DESCRIPTION
**Pull request by mapra99**
Closes #48 
This PR adds a new endpoint to create a new stock for an existing product.
```json
// GET /admin/v1/products/2/stocks
// Response body:
[
    {
        "id": 50,
        "product_id": 2,
        "quantity": 5,
        "toppings": {
            "color": {
                "id": 19,
                "value": "black",
                "price_change": "+100"
            },
            "size": {
                "id": 20,
                "value": "XS",
                "price_change": "+10"
            }
        }
    },
    {
        "id": 51,
        "product_id": 2,
        "quantity": 5,
        "toppings": {
            "color": {
                "id": 21,
                "value": "white",
                "price_change": "+100"
            },
            "size": {
                "id": 20,
                "value": "XS",
                "price_change": "+10"
            }
        }
    }
]
```

### CODE REVIEW

File: config/routes.rb [hunk 1]
```diff
@@ -7,9 +7,11 @@
   namespace :admin do
     namespace :v1 do
       resource :health, only: [:show], controller: :health
-      resources :products, only: %i[create index show]
       resources :product_categories, only: %i[create index]
       resources :product_contents, only: %i[create]
+      resources :products, only: %i[create index show] do
+        resources :stocks, only: %i[index]
+      end
```
Comment (ImMPrada) [inline config/routes.rb]: Ok, here, merging with PR 49, there will be something like:

```
resources :products, only: %i[create index show] do
        resources :stocks, only: %i[create index]
      end
end
```

Right?

Comment (mapra99) [reply config/routes.rb]: yup!

**Review by ImMPrada — APPROVED** [review]
Nice! I think just need to solve merging conflicts

File: config/routes.rb (hunk 1 above)
Comment (ImMPrada) [reply config/routes.rb]: yup.validateAt xD

**Review by ImMPrada — APPROVED** [review]

**Pull request merged**